### PR TITLE
Add possibility to disable sidecars for jobs and cronjobs

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Add possibility to disable sidecars for jobs and cronjobs

**Related issue(s)**

### CODE REVIEW

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -615,6 +615,7 @@ cron-jobs:
   create-modal:
     day-of-month: Day of Month
     day-of-week: Day of Week
+    disable-sidecar: Disable sidecar
```
Comment (Wawrzyn321) [inline core/src/i18n/en.yaml:618]: Maybe remove that translation and use the one from `jobs` section? Common stuff for Jobs and CronJobs is here anyway (see `ContainerSection.js`, which uses only `jobs.` prefixed translation keys and is used for both Jobs and CJs).

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
View breaks on entering the CronJob create modal
image

File: core-ui/src/components/Predefined/Create/Jobs/CronJobs.create.js [hunk 2]
```diff
@@ -29,15 +34,71 @@ function CronJobsCreate({
...
+                ...cronJob.spec.jobTemplate.spec.template,
+                metadata: {
+                  ...cronJob.spec.jobTemplate.spec.template.metadata,
+                  annotations: jobTemplateAnnotations,
+                },
+              },
+            },
+          },
+        },
+      });
```
Comment (Wawrzyn321) [inline core-ui/src/components/Predefined/Create/Jobs/CronJobs.create.js:85]: Maybe use
```
jp.value(cronJob, 'turbo-long-path', jobTemplateAnnotations);
setCronJob({...cronJob});
```
?

**Review by Wawrzyn321 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**